include/exadg/fluid_structure_interaction_precice/exadg_interface.h

```diff
@@ -3,6 +3,8 @@
 #include <deal.II/matrix_free/fe_evaluation.h>
 
 #include <exadg/fluid_structure_interaction_precice/coupling_interface.h>
+// TODO: Rename this file to exadg-terminal?
```

Comment:
It's good to have the TODO, but we should wait how things develop and whether we want to combine parts of this code with the current `ExaDG::InterfaceCoupling`, which could e.g. be realized as follow-up PRs of the present PR.

Reply:
Alright